src/componenets/navbar/navbar/Navbar.js

```diff
@@ -0,0 +1,15 @@
+import { NavLink } from 'react-router-dom';
+
+function Navbar() {
+  return (
+    <nav>
+      <NavLink to="/">Doctors</NavLink>
+      <NavLink to="/book-appointment">Book Appointment</NavLink>
+      <NavLink to="/my-appointments">My Appointments</NavLink>
+      <NavLink to="/add-doctor">Add Doctor</NavLink>
+      <NavLink to="/delete-doctor">Delete Doctor</NavLink>
+      <button type="button">Log Out</button>
+    </nav>
+  );
+}
+export default Navbar;
```

Comment:
Also for this may you please try to remove the extra navbar folder because I do not  know why  you are nesting navbar two times.
Could you change it to `src\components\navbar\Navbar.js`